### PR TITLE
Add Frostcrag Spire Cobl Alchemy Station

### PR DESCRIPTION
**Pull request by charabaruk**
Includes #1623

### CODE REVIEW

**Comment by charabaruk** [comment]
Oops forgot the reqs on this one! Don't merge yet.

**Comment by sibir-ine** [comment]
You can mark a PR as a draft if it's not ready to merge:
image

**Comment by charabaruk** [comment]
Is there a good way to specify the original DLCFrostcrag.esp and NOT Frostcrag Reborn's DLCFrostcrag.esp, by the way?

**Comment by sibir-ine** [comment]
If it's confirmed not to work with *Frostcrag Reborn*, you can mark them as incompatible using the checksum of the file from *Frostcrag Reborn*. The plugin already has `DLCFrostcrag.esp` as a master, so adding it again as a requirement would be redundant.

**Comment by charabaruk** [comment]
Okay, so no need to add a req item for masters? Then I just need to state Cobl and say to load after the unofficial patch. Plus of course mark incompatible with Frostcrag Reborn.

**Comment by charabaruk** [comment]
Actually I'm unsure if it conflicts with Frostcrag Reborn or not, since I don't use Frostcrag Reborn at all. I guess for now I'll leave out the incompat message, and if someone finds out down the road that they don't work together the message can get added then.

File: masterlist.yaml
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
```

The plugin already has Cobl as a master, so the requirement is redundant. `DLCFrostcrag - Unofficial Patch.esp` is in an earlier-loading group, so the after is also redundant. V1.0 of the plugin has the tag in its description, so that's also redundant. We also don't keep cleaning data for older versions since that'd bloat the masterlists.

File: masterlist.yaml
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
```

Version comments aren't necessary, the version number can be noted in the commit message.

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by charabaruk** [comment]
This should be good once #1623 is merged

**Comment by sibir-ine** [comment]
This PR needs to be rebased too, but I'd wait until #1622 is merged as it'll conflict with that too.

**Comment by charabaruk** [comment]
I had already gone to bed, I'll take care of rebasing right now before going to work.

EDIT: I will check during my breaks for #1622 getting merged, and rebase this again when that happens.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**
